src/main-oe/sgxlkl_run_oe.c

```diff
@@ -700,80 +835,25 @@ void get_signed_libsgxlkl_path(char* path_buf, size_t len)
     sgxlkl_host_fail("Unable to locate libsgxlkl.so.signed\n");
 }
 
-void set_sysconf_params(sgxlkl_config_t* conf, long ethreads_num)
+void mk_clock_res_string(int clock)
 {
-    conf->sysconf_nproc_conf = ethreads_num;
-    conf->sysconf_nproc_onln = ethreads_num;
+    sgxlkl_enclave_config_t* econf = &host_state.enclave_config;
+    size_t sz = sizeof(sgxlkl_clock_res_config_t);
+    struct timespec tmpt;
+    char tmps[8 * 2 * 2 + 1];
+    clock_getres(clock, &tmpt);
+    snprintf(tmps, sizeof(tmps), "%08lx%08lx", tmpt.tv_sec, tmpt.tv_nsec);
+    memcpy(econf->clock_res[clock].resolution, tmps, sz);
```

Comment:
`clock_res` is an attested setting, but will hopefully soon be removed.

Reply:
OK, since this is patched in release mode as well it would make the eeid mrenclave hard to predict. Can you add a TODO note that this will be removed or done differently?

Reply:
Yeah, same as the auto-detection of `fsgsbase`.